code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -151,12 +151,12 @@ LINEN BINS
 
 
 /obj/structure/bedsheetbin/update_icon()
-	switch(amount)
-		if(0)
-			icon_state = "linenbin-empty"
-		if(1 to amount / 2)
-			icon_state = "linenbin-half"
-		else				icon_state = "linenbin-full"
```

Comment:
>if `amount` is between 1 and half of `amount`

Yeah this one would really make you think even without the bug

Reply:
Yeeah.. lol.

Reply:
I don't really like using initial() but I'm not sure of a better way here besides making a new variable.

Reply:
I feel the same way about `initial()`. Making a new variable would be the only alternative, though. I don't really think anyone cares much either way about this particular object.

Reply:
If only someone had made a system that provided all objects with a `maxHealth` var.

Reply:
Are you saying this thing should treat its number of contained bedsheets as health?

Reply:
I was reading the other changes and didn't think too hard about this one. Yeah, this one is fine as it is, I guess.